custom/nutrition_project/ucr/data_sources/pregnancy_registrations_v1.json

```diff
@@ -0,0 +1,257 @@
+{
+  "domains": [
+    "india-nutrition-project"
+  ],
+  "server_environment": [
+    "india"
+  ],
+  "config": {
+    "table_id": "static-pregnancy_registrations_v1",
+    "display_name": "Pregnancy Registrations V1",
+    "referenced_doc_type": "XFormInstance",
+    "configured_filter": {
+      "type": "and",
+      "filters": [
+        {
+          "type": "boolean_expression",
+          "operator": "eq",
+          "expression": {
+            "type": "property_name",
+            "property_name": "xmlns"
+          },
+          "property_value": "http://openrosa.org/formdesigner/C598D396-C69A-4F1B-84FD-92B582DDC57D"
+        },
+        {
+          "type": "boolean_expression",
+          "operator": "eq",
+          "expression": {
+            "type": "property_name",
+            "property_name": "app_id"
+          },
+          "property_value": "b3c758f7c79e47e3a503115894f18503"
+        },
+        {
+          "type": "not",
+          "filter": {
+            "type": "boolean_expression",
+            "operator": "in",
+            "expression": {
+              "type": "property_path",
+              "property_path": [
+                "form",
+                "final_edd_calc"
```

Comment:
this would only allow those form which have this question answered to enter this datasource. Are other fields alone not used without final_edd_calc?

Reply:
Ya we only need to pick, only if the woman is actually pregnant. Since this is a "Pregnancy Registration" it should be true always ideally, So i am going to confirm on the need of this pre filter.

Reply:
Confirmed that that filter was not needed. Have removed it.

Reply:
thank you @Rohit25negi